app/services/forest_liana/has_many_dissociator.rb

```diff
@@ -14,20 +14,31 @@ def perform
 
       remove_association = !@with_deletion || @association.macro == :has_and_belongs_to_many
 
+      record_ids = Array.new
```

Comment:
[Question] Not really a convention in our Ruby code to instantiate variables that will be affected in the future.
Maybe you won't have issue to remove this if you use `if !record_ids.nil? && record_ids.any?` in the code below.